### DISCUSSION TITLE
Icons for Nodes

### DISCUSSION THREAD

**Discussion by johnnyjmcivorv** [zim-desktop-wiki/zim-desktop-wiki#2702] [Ideas]
CherryTree uses icons for the nodes in its tree browser. It really helps finding specific nodes at a glance. I think it would be cool if you could A) select from a box of emojis to use as node icons, and B) paste in a custom emoji or unicode character.

![image](https://github.com/user-attachments/assets/72dc93af-f6ba-4877-bc85-d2884ffdb840)

**Comment by osky85** [comment]
As a reference, this exits: https://github.com/Pl-M/zim-icontags-plugin

But it was last updated 7 years ago and does not work with current Zim version
